lib/apr/views/commerce_slack_view.ex

```diff
@@ -6,7 +6,15 @@ defmodule Apr.Views.CommerceSlackView do
     CommerceErrorSlackView
   }
 
+  alias Apr.Subscriptions.{
+    Subscription
+  }
+
   def render(event, routing_key) do
```

Comment:
i don't think we have case of render without subscription anymore, i think this file can have 2 `render` functions
```elixir
def render(%Subscription{theme: "fraud"}, event, routing_key) do
  # render fraud template
end
def render(_, event, routing_key) do
   # current implementation
end
```